code/stylesheet.dm

```diff
@@ -62,6 +62,10 @@ h1.alert, h2.alert		{color: #000000;}
 .rough					{font-family: 'Trebuchet MS', cursive, sans-serif;}
 .say_quote				{font-family: Georgia, Verdana, sans-serif;}
 .interface				{color: #330033;}
+.big					{font-size: 150%}
+.bold					{font-weight: bold;}

```

Comment:
There's no need for this as this should be done through <b> or <strong> unless you're wanting to make it bolder or boldest.